composer.json

```diff
@@ -10,14 +10,14 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": ">=7.4"
     },
     "require-dev": {
         "prestashop/php-dev-tools": "~3.0"
     },
     "config": {
         "platform": {
-            "php": "5.6.0"
```

Comment:
You must keep these lines as ps_cashondelivery is always compatible with PS 1.7.6.